IdentityCore/src/oauth2/aad_base/MSIDAADWebviewFactory.m

```diff
@@ -84,8 +108,10 @@ @implementation MSIDAADWebviewFactory
     result[@"haschrome"] = @"1";
     [result addEntriesFromDictionary:MSIDDeviceId.deviceId];
     
+    NSString* tenantId = parameters.accountIdentifier.utid;
```

Comment:
nit: Our code style is: `NSString *tenantId = ...` (the `*` goes next to the variable name)